crafters/nlp/PDFExtractorSegmenter/tests/test_pdfextractorcrafter.py

```diff
@@ -0,0 +1,38 @@
+from .. import PDFExtractorSegmenter
+from PIL import Image
+import os
+
+expected_text = "A cat poem\nI love cats, I love every kind of cat,\nI just wanna hug all of them, but I can't," \
+                    "\nI'm thinking about cats again\nI think about how cute they are\nAnd their whiskers and their " \
+                    "nose\n "
+
+def test_io_uri():
+    crafter = PDFExtractorSegmenter()
+    chunks = crafter.craft(uri='cats_are_awesome.pdf', buffer=None)
+
+    # Check image
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    img = Image.open(os.path.join(cur_dir, '../test_img.jpg'))
+    blob = chunks[0]['blob']
+    assert img.width == blob.width
+    assert img.height == blob.height
```

Comment:
it would be cool to check if the image is the same as the blob, you can check the tests of `TorchObjectDetectionSegmenter` for reference

Reply:
both for chunk[0] and chunk[1]